feat: add support for JWT connexion on GoogleBigQuery connector [TCTC-6756] (#1287)
ToucanToco/toucan-connectors: doc/connectors/google_big_query.md

* `token_uri`: str
* `auth_provider_x509_cert_url`: str
* `client_x509_cert_url`: str
* `jwt_token`: str

> You can also authenticate with a `signed jwt_token` you've created yourself,

Reviewer: ```suggestion
You can also authenticate with a signed  `jwt_token` you've created yourself,
```

in that case, you will only need here two fields:

- `project_id`: str
- `jwt_token`: str

> *Note*: you will be responsible on updating the `jwt_token` when it expired or
> set a hight value for the expiration.

Reviewer: probably not the best idea to suggest setting a high expiration 😄 
```suggestion
*Note*: you will be responsible of updating the `jwt_token` when it expires
```
